### DISCUSSION TITLE
Could I use clangd for arm64?

### DISCUSSION THREAD

**Discussion by wss29** [clangd/clangd#2137] [Q&A]
Hi,
   It seem that [clangd-linux-18.1.3.zip](https://github.com/clangd/clangd/releases/download/18.1.3/clangd-linux-18.1.3.zip) can not be used on arm64, could clangd be used on arm64 arch？

**Comment by HighCommander4** [comment]
Please see https://github.com/clangd/clangd/issues/2009#issuecomment-2053684883
